### PR TITLE
add organisations to vocab and add xslt to Norsk Profil ISO

### LINKED ISSUE #252: Create a new xslt for Norsk Profil iso
A more general xslt should be developed to support more general translation to Norsk Profil of ISO used on genorge. This can replace mmd-to-genorge after testing, but it will be used in a dedicated csw endpoint. 

- create mmd-to-Nois.xsl
- add Collection to vocabulary to include Organizations names in English and Norwegian

### PR DESCRIPTION
**Pull request by ferrighi**
Closes #252 

- Added Organisation mapping to vocab (to be used to provide Norwegian names to genorge)
- Added dedicated mmd-to-Noiso to generalise mmd-to-genorge

### CODE REVIEW

**Comment by ferrighi** [comment]
I've found an issue on the xslt, for a special case. I'll fix this first.

**Comment by ferrighi** [comment]
The issues of duplicate personnel should be fixed now.

File: thesauri/mmd-vocabulary.ttl [hunk 1]
```diff
@@ -1387,3 +1387,37 @@
   skos:prefLabel "Comprehensive quality control"@en ;
   skos:definition "The dataset has undergone extensive quality control, including but not limited to basic and advanced automatic controls and regular manual control. The result of the quality control follows the data as flags."@en .
 
+<https://vocab.met.no/mmd/Organisation>
+  a skos:Collection, isothes:ConceptGroup> ;
+  skos:inScheme <https://vocab.met.no/mmd> ;
+  skos:prefLabel "Organisation"@en ;
+  skos:definition "A controlled vocabulary for the organisations names in English and Norwegian"@en ;
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.ttl:1394]:
```suggestion
  skos:definition "A controlled vocabulary for the organisation names in English and Norwegian"@en ;
```

File: thesauri/mmd-vocabulary.xml [hunk 2]
```diff
@@ -1837,4 +1837,60 @@
     </skos:member>
 
   </skos:Collection>
+
+  <skos:Collection rdf:about="https://vocab.met.no/mmd/Organisation">
+    <rdf:type rdf:resource="http://purl.org/iso25964/skos-thes#ConceptGroup"/>
+    <skos:inScheme rdf:resource="https://vocab.met.no/mmd"/>
+    <skos:prefLabel xml:lang="en">Organisation</skos:prefLabel>
+    <skos:definition xml:lang="en">A controlled vocabulary for the organisations names in English and Norwegian</skos:definition>
```
Comment (mortenwh) [inline thesauri/mmd-vocabulary.xml:1845]:
```suggestion
    <skos:definition xml:lang="en">A controlled vocabulary for organisation names in English and Norwegian</skos:definition>
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
LGTM - except two language corrections

**Comment by ferrighi** [comment]
Thanks! I've made the changes.

**Pull request merged**